lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+{parseString} = require 'xml2js'
+path = require 'path'
+
+module.exports =
+  config:
+    executablePath:
+      title: 'CSSLint Executable Path'
+      type: 'string'
+      # default: path.join(__dirname, '..', 'node_modules', '.bin', 'csslint.cmd')
```

Comment:
On that note, currently on both my systems I had to manually specify the path, since even if you point it at the bundled directory it's not smart enough to use PATHEXT.

I'll look into whether cli.js can be used.